spec/system/user_management_spec.rb

```diff
@@ -13,10 +13,10 @@
     end
     it 'puts the errors in the flash' do
       visit users_url
-      # Find destroy link by href because there are 2 users by necessity.
-      # Although you definitely _can_ delete yourself.
-      find(:xpath, "//a[@href='/users/#{@user.id}']").click
-      page.driver.browser.switch_to.alert.accept
+      # Click the 'destroy' link in the user's row
+      page.accept_confirm 'Are you sure?' do
+        within('tr', text: @user.name) { click_on 'Destroy' }
```

Comment:
It was this easy? I knew there must be a way to do it but I admit, I was too lazy to look it up at the time and thought it must be more impenetrable than finding the button by xpath.